chia/rpc/wallet_rpc_api.py

```diff
@@ -1696,6 +1696,10 @@ async def create_signed_transaction(self, request, hold_lock=True) -> EndpointRe
         if "coins" in request and len(request["coins"]) > 0:
             coins = set([Coin.from_json_dict(coin_json) for coin_json in request["coins"]])
 
+        exclude_coins = None
+        if "exclude_coins" in request and len(request["exclude_coins"]) > 0:
+            exclude_coins = set([Coin.from_json_dict(coin_json) for coin_json in request["exclude_coins"]])
```

Comment:
exclude_coins is a list, not a set.

Reply:
`exclude_coins` gets passed into `generate_signed_transaction()` which accepts a set, or do you mean something else?

Reply:
sorry i mixed up the api and the client when reviewing